src/components/HistoryTables/defaultCacheConfig.ts

```diff
@@ -16,8 +15,6 @@ const parseParams = (_: any, incoming: any) => {
     }
     return null
   } catch (e) {
-    const msg = e instanceof Error ? e.message : String(e)
```

Comment:
@navFooh i couldn't think of a great way to replicate the logger here... i'm not convinced we need to log errors here though?